### DISCUSSION TITLE
p/0f59778854f14a06aa63658b8bca8691/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#54] [Announcements]
# p/0f59778854f14a06aa63658b8bca8691/

又过了接近两周，东三省和内蒙的旅程已经结束，吾辈正在前往拉萨的火车上写下了这篇内容。东三省是一个尽管经济落后，但铁路网络相当发达的地区。相比于西北地区到哪儿都要报团拼车之类的，东三省的许多看起来很小的地方也都通了火车。这次东三省之旅吾辈几乎没有选择大城市，除了下飞机落地的大连之外，都是一些很小的城市：丹东、长白山和抚远。内蒙则选择了海拉尔，现在也叫呼伦贝尔，大草原就在旁边，通着火车太棒了。  大连

https://blog.rxliuli.com/p/0f59778854f14a06aa63658b8bca8691/

**Comment by kinoward** [comment]
大佬真的非常厉害👍，行动力 max，摄影也非常棒，想问一下大佬是用什么设备拍摄的？

**Reply by rxliuli** [reply]
nikon z50 + nikon z 28-400mm

**Reply by kinoward** [reply]
非常感谢，我也参考一下

**Comment by observerkk** [comment]
拍的好好看
